ui/routes.py

```diff
@@ -26,7 +26,7 @@ def handle_api_exceptions(ex: Exception, req: falcon.Request, _resp: falcon.Resp
     if isinstance(ex, (falcon.HTTPNotFound, falcon.HTTPNotImplemented)):
         raise falcon.HTTPPermanentRedirect(location="/404")
 
-    if isinstance(ex, (falcon.HTTPPermanentRedirect, falcon.HTTPTemporaryRedirect)):
+    if isinstance(ex, (falcon.HTTPPermanentRedirect, falcon.HTTPSeeOther)):
```

Comment:
Please do not replace the Temporary redirect exception. Add the SeeOther exception as another option to the list

Reply:
Ok. But why isn't `falcon.HTTPTemporaryRedirect` handled alongside `falcon.HTTPPermanentRedirect` in `_handle_exceptions` at line 47?